lxd/auth/authorization.go

```diff
@@ -0,0 +1,73 @@
+package auth
+
+import (
+	"net/http"
+
+	"github.com/canonical/lxd/lxd/request"
+	"github.com/canonical/lxd/shared"
+)
+
+type Authorizer interface {
+	AddProject(id int64, name string) error
```

Comment:
can we clarify what the `id` represents in these functions?

Reply:
I've renamed it to `projectID`.

Reply:
Thanks. Does this function check whether the user can add a project? If so can we change the name to CanAddProject?

Reply:
No, this doesn't check any permissions. It's used for letting RBAC know about new projects.